docs/Troubleshooting.md

```diff
@@ -0,0 +1,79 @@
+# Get verbose logs
```

Comment:
the troubleshooting is nice to have! My only doubt Is if is better to have it just in the newrelic docs or here as well. What you had in mind?

Reply:
the idea is to move all this docs to the NR docs page for public preview/GA.